openstack/standalone-cinder/pkg/provisioner/vsbroker.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioner
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/kubernetes-incubator/external-storage/lib/controller"
+	"github.com/kubernetes-incubator/external-storage/openstack/standalone-cinder/pkg/volumeservice"
+)
+
+type volumeServiceBroker interface {
+	CreateCinderVolume(vs *gophercloud.ServiceClient, options controller.VolumeOptions) (string, error)
```

Comment:
nit: godoc

Reply:
This is an interface used just for overriding cinder calls in the volumeservice module.  I'll add a comment about the interface at the top.